csrf.go

```diff
@@ -104,23 +103,31 @@ type Options struct {
 	ErrorFunc func(w http.ResponseWriter)
 }
 
+const (
+	letterBytes   = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"
+	letterIdxBits = 6                    // 6 bits to represent a letter index
+	letterIdxMask = 1<<letterIdxBits - 1 // All 1-bits, as many as letterIdxBits
+	letterIdxMax  = 63 / letterIdxBits   // # of letter indices fitting in 63 bits
+)
```

Comment:
I don't see a reason why they should live outside the `randomBytes` function, as they are only ever being used within that function, please move them in.